main.nf

```diff
@@ -694,15 +742,108 @@ if( params.bt_index ) {
 } else{
     ch_genome_bam_flagstat_mqc = Channel.empty()
 }
+/*
+ * STEP 7.0, 7.1 and 7.2 if skip_mirdeep is not specified
+ */
+if (! params.skip_mirdeep){
```

Comment:
Is there a reason why don't use the `when` directive for all three processes? This works, but code might look cleaner from a nextflow perspective.

Reply:
Yes, no problem, I can change it!

Reply:
I have changed it, but it doesn't seem to work correctly, at least for the tests that I'm doing. I have also noticed that the `--skipMultiqc` doesn't seem to work